src/Ilicop.Web/Contracts/SettingsResponse.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
-namespace Geowerkstatt.Ilicop.Web
+namespace Geowerkstatt.Ilicop.Contracts
```

Comment:
REQ: Namespaces korrekt nach Default Namespace richten.
```suggestion
namespace Geowerkstatt.Ilicop.Web.Contracts
```